### ISSUE TITLE
VxMark: volume & rate changes are not spoken back to the user if something else is being spoken

### ISSUE THREAD

**Issue by arsalansufi** [votingworks/vxsuite#7676]
*Problem:* 
Not sure if this is a bug or intended behavior - On both VxMark and VxMarkScan if you press a button to change volume level and/or rate while some other text is being spoken, the text is NOT interrupted and there is never audio played about the volumne/rate changing, it just changes as the rest of whatever was previously playing continues.

*From:* Caroline

*Proposed solution:* 

_Collected by @caroline_

Issue automatically created from Slack feedback reported by Product Feedback: 
https://votingworks.slack.com/archives/CJU9MSC6S/p1765231211581589

**Comment by adghayes** [comment]
we decided that's intended behavior
